### DISCUSSION TITLE
Input on "set CFG Scale" does not maintain decimal precision and causes error when enabling/saving Automatic1111 image generation

### DISCUSSION THREAD

**Discussion by VirgilVasiu** [open-webui/open-webui#7355] [Issues]
# Bug Report

---

## Installation Method
Python using pipx install open-webui

## Environment

- **Open WebUI Version:** v0.4.4
- **Ollama (if applicable):** 0.4.2

- **Operating System:** Debian 12.2.0-14
- **Browser (if applicable):** Chromium 131.0.6778.85

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Enable Image Generation (Experimental) and save to enable image generation functionality with Automatic1111.

## Actual Behavior:

When clicking save the webpage freezes and nothing can be done until page is reloaded. Then page shows changes but image generation has not been enabled. 
Log of open-webui shows the following error: INFO:     192.168.X.XX:4515 - "POST /images/api/v1/config/update HTTP/1.1" 422 Unprocessable Entity

## Description

**Bug Summary:**
Input on "set CFG Scale" does not maintain decimal precision and causes error when enabling/saving Automatic1111 image generation settings.

## Reproduction Details

**Steps to Reproduce:**
Inputting 7.0 into the "set CFG Scale" field and enabling image generation and saving will allow image generation to work with no issues. However if any change is made or the page refreshed for any reason then the 7.0 will become 7 and saving will cause the webpage to freeze and the changes to not be saved.

If 7.1 is used then no issues occur, and reloading the page keeps the 7.1 populated so saving changes works properly.

## Additional Information

Issue was discussed and has screenshots in https://github.com/open-webui/open-webui/discussions/5976

**Comment by tjbck** [comment]
Unable to reproduce this issue, keep us updated!
![image](https://github.com/user-attachments/assets/dc1ed9ef-ab86-4a9b-9a39-382b62549370)

**Comment by TheMasterFX** [comment]
Same here. Everytime I want to change something, I MUST also reenter the CFG Scale (even if I don't change the value). Else the interface hangs after clicking the Save-Button. Then you have to refresh Open WebUI and do it again (but this time reenter the CFG)
I'm using 0.48
